packages/experimental/src/rest/types.ts

```diff
@@ -37,4 +37,18 @@ export interface RestEndpoint<
   ) => any;
   method: string;
   signal: AbortSignal | undefined;
+  paginated?: (...args: any) => any;
```

Comment:
it really doesn't matter what return type this function has??

Reply:
made it more specialized